.github/workflows/scripts/gcp-get-cached-disks.sh

```diff
@@ -43,31 +43,29 @@ find_cached_disk_image() {
 # Check if both $DISK_PREFIX and $DISK_SUFFIX are set, as they are required to find a cached disk image
 if [[ -n "${DISK_PREFIX}" && -n "${DISK_SUFFIX}" ]]; then
     # Find the most suitable cached disk image
-    echo "Finding the most suitable cached disk image..."
+    echo "Finding a ${DISK_PREFIX}-${DISK_SUFFIX} disk image for ${NETWORK}..."
     CACHED_DISK_NAME=""
 
-    # First, try to find a cached disk image from the current commit
-    CACHED_DISK_NAME=$(find_cached_disk_image ".+-${GITHUB_SHA_SHORT}" "commit")
-
-    # If no cached disk image is found
-    if [[ -z "${CACHED_DISK_NAME}" ]]; then
-        # Check if main branch images are preferred
-        if [[ "${PREFER_MAIN_CACHED_STATE}" == "true" ]]; then
-            CACHED_DISK_NAME=$(find_cached_disk_image "main-[0-9a-f]+" "main branch")
-        # Else, try to find one from any branch
-        else
+    # Check if main branch images are preferred
+    if [[ "${PREFER_MAIN_CACHED_STATE}" == "true" ]]; then
+        CACHED_DISK_NAME=$(find_cached_disk_image "main-[0-9a-f]+" "main branch")
+    # Else, try to find a cached disk image from the current branch (or PR)
+    else
+        CACHED_DISK_NAME=$(find_cached_disk_image ".+-${GITHUB_REF}" "branch")
```

Comment:
Shouldn't we test if `GITHUB_REF` isn't empty before we pass it to `find_cached_disk_image`?

Reply:
I think the only benefit will be that we'd be able to `echo` an error in the logs. But if the `GITHUB_REF` is missing, then there's a bunch of other things that will also fail in other workflows, as there's not supposed to be a commit without a reference.

Unless you're thinking in other future scenarios with different ways to call this script.